src/examples/chrome/ChromeNavigation.tsx

```diff
@@ -0,0 +1,141 @@
+
+  return (
+    <Row>
+      <Col>
+        <Chrome isFluid style={{ height: 500 }} hue={PALETTE.blue[800]}>
+          <Nav isExpanded={expanded}>
+            <NavItem hasLogo>
+              <NavItemIcon>
+                <ProductIcon />
+              </NavItemIcon>
+              <NavItemText>Zendesk Support</NavItemText>
+            </NavItem>
+            <NavItem isCurrent={nav === 'nav-1'} onClick={() => setNav('nav-1')}>
+              <NavItemIcon>
+                <HomeIcon />
+              </NavItemIcon>
+              <NavItemText>Home</NavItemText>
+            </NavItem>
+            <NavItem isCurrent={nav === 'nav-2'} onClick={() => setNav('nav-2')}>
+              <NavItemIcon>
+                <EmailIcon />
+              </NavItemIcon>
+              <NavItemText>Email</NavItemText>
+            </NavItem>
+            <NavItem isCurrent={nav === 'nav-3'} onClick={() => setNav('nav-3')}>
+              <NavItemIcon>
+                <SettingsIcon />
+              </NavItemIcon>
+              <NavItemText>Settings</NavItemText>
+            </NavItem>
+            <NavItem hasBrandmark title="Zendesk">
+              <NavItemIcon>
+                <ZendeskIcon />
+              </NavItemIcon>
+              <NavItemText>Zendesk</NavItemText>
+            </NavItem>
+          </Nav>
+          {showSubNav && (
+            <SubNav>
+              <SubNavItem isCurrent={subNav === 'item-1'} onClick={() => setSubNav('item-1')}>
+                <SubNavItemText>Subnav 1</SubNavItemText>
+              </SubNavItem>
+              <SubNavItem isCurrent={subNav === 'item-2'} onClick={() => setSubNav('item-2')}>
+                <SubNavItemText>Subnav 2</SubNavItemText>
+              </SubNavItem>
+              <CollapsibleSubNavItem
+                header="Collapsible Item"
+                onChange={setCollapsed}
+                isExpanded={collapsed}
+              >
+                <SubNavItem
+                  isCurrent={subNav === 'collapsed-item-1'}
+                  onClick={() => setSubNav('collapsed-item-1')}
+                >
+                  <SubNavItemText>Item 1</SubNavItemText>
+                </SubNavItem>
+                <SubNavItem
+                  isCurrent={subNav === 'collapsed-item-2'}
+                  onClick={() => setSubNav('collapsed-item-2')}
+                >
+                  <SubNavItemText>Item 2</SubNavItemText>
+                </SubNavItem>
+                <SubNavItem
+                  isCurrent={subNav === 'collapsed-item-3'}
+                  onClick={() => setSubNav('collapsed-item-3')}
+                >
+                  <SubNavItemText>Item 3</SubNavItemText>
+                </SubNavItem>
+              </CollapsibleSubNavItem>
+              <SubNavItem isCurrent={subNav === 'item-3'} onClick={() => setSubNav('item-3')}>
+                <SubNavItemText>Subnav 3</SubNavItemText>
+              </SubNavItem>
+            </SubNav>
+          )}
+          <Body>
```

Comment:
A `<Header />` component must be added – even if empty. Without it, we get the unexpected space showing beneath the content in this example.